app/services/source_refresh_service.rb

```diff
@@ -31,6 +37,9 @@ def process
 
         if persister_task.state == "completed"
```

Comment:
@hsong-rh Can this persister_task also be cleared by a cronjob, when the relationship is loaded will it fail?